### DISCUSSION TITLE
use_force_sensor

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#30] [Q&A] [answered]
スライドにはuse_force_sensorをfalseにすると力センサを全く使わなくなるとありますが、実際には脚では使っている?
https://github.com/YutaKojio/hrpsys-base/blob/5aab7f9809b1ef76dd62bf323e522834d84097be/rtc/AutoBalancer/Stabilizer.cpp#L453

**Comment by YutaKojio** [comment, marked as answer]
== std::string::npos なので腕だけかな．
確かに語弊がありました．use_force_sensor は足に関してだけです．
